### PR TITLE
Add ModPage, and ModListPage

### PR DESCRIPTION
**Pull request by ppapman1**
ok this is a very good thing
and my code looks like spaghetti
plz help me

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -24,11 +24,14 @@ function App() {
       <AppRouter />
 
       {/* FOOTER */}
-      <footer>
-        <div className="footer-content">
-          <h4 style={{ fontWeight: '400' }}>ADOFAI.GG is not associated with <a href="https://7thbe.at/" target="_blank" rel="noreferrer">7th Beat Games</a>.&ensp;|&ensp;<a href="https://discord.gg/rwWyfPJ" target="_blank" rel="noreferrer">Join Our Discord</a>&ensp;|&ensp;<a href="mailto:adofai.gg@gmail.com">Contact Us</a></h4>
-        </div>
-      </footer>
+      {!window.location.pathname == '/mods' ? (
```
Comment (ppapman1) [inline src/components/App.js:27]:
```suggestion
      {window.location.pathname !== '/mods' ? (
```
fix error(or typo)

File: src/routes/ModListPage.js [hunk 2]
```diff
@@ -1,13 +1,125 @@
...
+    }, 500);
+  };
+
   return (
     <>
-      <div className="mods-list-main">
-        attachments
+      <div className="mods-main">
+        <SearchSection
+          placeholder='Search to Mod Name'
```
Comment (ppapman1) [inline src/routes/ModListPage.js:32]:
```suggestion
          placeholder='Search Mod'
```
better than 'search to mod name'

File: src/components/ModsArticle.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<div className="mods-list-item">
```
Comment (ppapman1) [inline src/components/ModsArticle.js:1]: LOL THIS FILE IS NOT USED

File: src/stylesheets/basic.css [hunk 4]
```diff
@@ -220,6 +220,127 @@ main {
...
+
+.list-search-content-title {
+  width: 1080px;
+  border-top-color: rgba(255, 255, 255, 0.4);
+  border-top-width: 1px;
+  border-top-style: solid;
+  padding-top: 10px;
+  font-size: 1.4em;
+  font-weight: 700;
+}
```
Comment (ppapman1) [inline src/stylesheets/basic.css:306]:
```suggestion
  margin-bottom: 5px;
}
```
i think it'd be nice to have a margin-bottom here

File: src/routes/ModListPage.js [hunk 5]
```diff
@@ -1,13 +1,125 @@
...
+            </form>
+          }
+        />
+
+        <div className="mods-list">
+        <InfiniteScroll
+          dataLength={items.length}
+          next={fetchMoreData}
+          hasMore={hasMore}
+          loader={<h4 style={{textAlign: 'center'}}>Wait a second please!</h4>}
```
Comment (ppapman1) [inline src/routes/ModListPage.js:78]:
```suggestion
          loader={<h4 style={{ textAlign: 'center', fontWeight: '300', marginTop: '10px', marginBottom: '10px' }}>Wait a second please!</h4>}
```
ok now this code is very longer
i havent tested it to see if its working
somebody check it out

**Review by ppapman1 — COMMENTED** [review]
good

**Comment by CrackThrough** [comment]
c

**Comment by ppapman1** [comment]
yay i think it is done

File: src/routes/ModListPage.js (hunk 5 above)
Comment (ppapman1) [reply src/routes/ModListPage.js]: fixed

File: src/stylesheets/basic.css (hunk 4 above)
Comment (ppapman1) [reply src/stylesheets/basic.css:302]: fixed

File: src/components/ModsArticle.js (hunk 3 above)
Comment (ppapman1) [reply src/components/ModsArticle.js]: fixed

File: src/routes/ModListPage.js (hunk 2 above)
Comment (ppapman1) [reply src/routes/ModListPage.js]: fixed

File: src/components/App.js (hunk 1 above)
Comment (ppapman1) [reply src/components/App.js]: fixed

**Review by cxhuy — APPROVED** [review]
Looks good to me

**Pull request merged**